src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -4387,7 +4406,7 @@ private void AddSynthesizedTupleMembersIfNecessary(MembersAndInitializersBuilder
 
                             AddAccessorIfAvailable(builder.NonTypeMembers, property.GetMethod);
                             AddAccessorIfAvailable(builder.NonTypeMembers, property.SetMethod);
-                            FieldSymbol backingField = property.BackingField;
+                            FieldSymbol backingField = property.NonFieldKeywordBackingField;
```

Comment:
>NonFieldKeywordBackingField [](http://example.com/codeflow?start=64&length=27)

I think this change should be reverted, assuming that we are reverting behavior of the ```BackingField``` property. When we start working on supporting initializers and mixed scenarios, we'll get back to this.

Reply:
Fixed.